shuup/simple_supplier/module.py

```diff
@@ -63,3 +65,5 @@ def update_stock(self, product_id):
         if latest_event:
             sv.stock_value_value = latest_event.purchase_price_value * sv.logical_count
         sv.save(update_fields=("logical_count", "physical_count", "stock_value_value"))
+        product = Product.objects.get(pk=product_id)
+        stock_updated.send(sender=type(self), supplier=self.supplier, product=product)
```

Comment:
Since this is SimpleSupplier specifig feature there is no need to send this signal. And it shouldn't be sent by simple supplier.

Instead you just can do the alert limit check here and send the notify event if needed.